data/companies.json

```
@@ -336,6 +336,22 @@
         "youtube": "https://www.youtube.com/c/Eminenceways"
       }
     },
+    {
+      "name": "e-Zone International",
+      "establishedAt": 1049155200,
+      "location": "Mahalaxmisthan, Lalitpur, Nepal",
```

Since the entire list is about companies from Nepal, we don't really need to mention Nepal in the address
```suggestion
      "location": "Mahalaxmisthan, Lalitpur",
```